scripts/panelite_dependencies.json

```diff
@@ -1,5 +1,68 @@
 {
     "developer_guide/Developing_Custom_Models.ipynb": [],
+    "gallery/apis/stocks_altair.ipynb": ["panel"],
```

Comment:
These are about to basically all go away (or rather move under how_to) so I'll rework this.